components/TableOfContent.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export interface TableContentModel {
+    id: string;
+    depth: number;
+    value: string;
+    key: string;
+}
+
+const TableOfContent = ({ titles }: { titles: TableContentModel[] }): JSX.Element => {
+    return (
+        <ul className="list-none p-0">
+            {titles.map((title) => {
+                return (
+                    <li key={title.key} className={classNames(`pl-${title.depth}`)}>
+                        {title.id.length ? (
+                            <a className="font-roboto text-sm text-gray" href={`#${title.id}`}>
```

Comment:
Why we need `font-roboto` and `text-sm` here?

Reply:
We don't really need that. I assumed the styles were going to be similar to sidenav but I will remove them.